js/app/modules/selection/featuredFirst.js

```diff
@@ -21,13 +30,11 @@ const AllHighlights = new Module.Class({
         if (query_index == 0) {
             return Eknc.QueryObject.new_from_props({
                 limit: limit,
-                tags_match_all: ['EknArticleObject', 'EknFeaturedTag'],
-                excluded_tags: [],
+                tags_match_all: ['EknFeaturedTag'],
```

Comment:
No, because when this query returns nothing, the `query_index` is incremented, and the second query excluding `EknFeaturedTag` kicks in.